tools/topology/topology2/include/components/multiband_drc/passthrough.conf

```diff
@@ -0,0 +1,71 @@
+# Exported Control Bytes 16-Aug-2023
```

Comment:
We really need to have a comment explaining how these are generated.

Reply:
Yep, I'll do a PR to change the text in the tool.

Reply:
Now fixed in this PR, re-exported the blobs. Tool PR is #8142.